torchvision/prototype/datasets/_builtin/coco.py

```diff
@@ -120,6 +120,45 @@ def _decode_instances_anns(self, anns: List[Dict[str, Any]], image_meta: Dict[st
             ann_ids=[ann["id"] for ann in anns],
         )
 
+    def _decode_person_keypoints_anns(self, anns: List[Dict[str, Any]], image_meta: Dict[str, Any]) -> Dict[str, Any]:
+        image_size = (image_meta["height"], image_meta["width"])
+
+        keypoints_meta = torch.tensor([ann["keypoints"] for ann in anns]).view(len(anns), -1, 3)
+        coordinates = keypoints_meta[..., :2]
+        visibility = Feature(keypoints_meta[..., 2])
+
+        # As of COCO 2017, only the person category has keypoints.
+        descriptions = (
+            "nose",
+            "left_eye",
+            "right_eye",
+            "left_ear",
+            "right_ear",
+            "left_shoulder",
+            "right_shoulder",
+            "left_elbow",
+            "right_elbow",
+            "left_wrist",
+            "right_wrist",
+            "left_hip",
+            "right_hip",
+            "left_knee",
+            "right_knee",
+            "left_ankle",
+            "right_ankle",
+        )
+        symmetries = [
+            ("vertical", description, description.replace("left", "right"))
+            for description in descriptions
+            if description.startswith("left")
+        ]
```

Comment:
Could you clarify the type of the resulting `symmetries` variable?

Below they `Keypoint` expects `Sequence[Tuple[KeypointSymmetry, Tuple[int, int]]]`. It's non obvious to me how the current code has the specific type.